backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewCreateService.java

```diff
@@ -239,4 +240,25 @@ public static boolean isEmptyObjectiveRequirement(Element self) {
                 .map(ObjectiveMembership.class::cast)
                 .findFirst().isEmpty();
     }
+
+    public Element createPartUsageAsAllocationDefinitionEnd(Element self, Element endParent) {
+        if (self instanceof AllocationDefinition) {
```

Comment:
My be the parameter self could by an AllocationDefinition instead of an Element. what do you think about?

Reply:
Done